src/ui-lib/sass/03-elements/_section.scss

```diff
@@ -1,6 +1,23 @@
 section {
   margin-top: 0;
 
+  p {
```

Comment:
Hmmm... the code below is mostly fine. However, I'd like the `max-width` value to be extracted as a variable and put under `settings/` somewhere. Likewise, the `margin-left` value ought to use one of the ones defined in `_spacing.scss`.

_However_, I wonder if a better solution is simply to adjust the max-width of our page content container. That way everything will be constrained to a sensible line-length. We can avoid needing to indent the heading by an arbitrary amount then too.

Having poked around the code, I think it may be worth revisiting that and the breakpoints. They're already being used somewhat inconsistently. Let's discuss with the wider team before proceeding.

Reply:
I'm ok to move the value into the typography css as `$grav-line-max-width`. I didn't assign `max-width` to the container because that would make everything narrower, even on designs where we don't want to use it, like locations or careers.

By applying a fixed margin left to the `hx` I ensured that the paragraphs are always going to be correctly aligned and that there is not going to be too much distance between a section title and its content.

Reply:
Just a thought. What about a new class that is going to take care about settings these rules?

Wrapping headers, paragraphs and other tags in a div is not that bad, it's what they are made for. That div will only have styling properties.


Example: